packages/block-editor/src/components/url-input/README.md

```diff
@@ -125,6 +125,14 @@ Renders the URL input field used by the `URLInputButton` component. It can be us
 }
 ```
 
+### `label: String`
+
+*Optional.* If this property is added, a label will be generated using label property as the content.
+
+### `id: String`
```

Comment:
We are documenting an id property but the id is not being used, should we remove this property from the documentation? The component seems to generate its own id based on the instance id.